jekyll/_cci2/language-python.md

```diff
@@ -55,27 +51,16 @@ The Python [orb]({{site.devhub_base_url}}/orbs/orb/circleci/python) contains a s
 To add the orb to your config, insert:
 ```yaml
 orbs:
-  python: circleci/python@1.5.0
-```
-
-Note: You might need to enable organization settings to allow the use of third-party orbs in the CircleCI dashboard, or request permission from your organization’s CircleCI admin.
-
-### 3. Create a workflow
-{: #create-a-workflow }
-
-A workflow is a set of rules for defining a collection of jobs and their run order. Workflows support complex job orchestration using a set of configuration keys to help you resolve failures sooner. Inside the workflow, you define the jobs you want to run. CircleCI will run this workflow on every commit. Learn more about [workflow configuration]({{ site.baseurl }}/2.0/configuration-reference/#workflows).
-
-```yaml
-workflows:
-  my_workflow: # This is the name of the workflow, feel free to change it to better match your workflow.
+  python: circleci/python@2.0.3
 ```
+**Note**: It is always a good idea to check in the [Orb Registry](https://circleci.com/developer/orbs) to ensure you are using the most recent (or correct for your project) orb verison.
```

Comment:
Maybe "or project specific" instead of "or correct for your project"? Can't put my finger on it, but it sounds odd.

Reply:
Yes agreed I did struggle with getting this to sound right